### PR TITLE
Add `sass-parser` support for the `@use` rule

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.79.7
```
Comment (ntkme) [inline CHANGELOG.md:1]: Not sure if it is intended, 1.79.6 has not been released.

Comment (nex3) [reply CHANGELOG.md:1]: Ah, thanks

**Review by Goodwine — CHANGES_REQUESTED** [review]

File: pkg/sass-parser/lib/src/configuration.ts [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+ * The set of raws supported by {@link Configuration}.
+ *
+ * @category Statement
+ */
+export interface ConfigurationRaws {
+  /** Whether the final variable has a trailing comma. */
+  comma?: boolean;
+
+  /**
+   * The whitespace between the final variable (or its trailing comma if it has
```
[the remark this answers is not in the document]
Comment (nex3) [reply pkg/sass-parser/lib/src/configuration.ts:28]: Yes, it's stored in `ConfiguredVariableRaws.afterValue`.

File: pkg/sass-parser/lib/src/statement/use-rule.ts [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        ? this.raws.url!.raw
+        : new StringExpression({text: this.useUrl, quotes: true}).toString();
+    const hasConfiguration = this.configuration.size > 0;
+    if (this.raws.namespace?.value === this.namespace) {
+      result += this.raws.namespace?.raw;
+    } else if (!this.namespace) {
+      result += ' as *';
+    } else if (this.defaultNamespace !== this.namespace) {
+      result += ' as ' + sassInternal.toCssIdentifier(this.namespace);
+    }
```
[the remark this answers is not in the document]
Comment (nex3) [reply pkg/sass-parser/lib/src/statement/use-rule.ts:127]: Yes, `myUse.raws.namespace` is strictly ignored if it doesn't match.

File: pkg/sass-parser/lib/src/configuration.ts [hunk 4]
```diff
@@ -114,7 +114,11 @@ export class Configuration extends Node {
     this._variables.clear();
   }
 
-  /** Removes the variable named {@link name} from this configuration. */
+  /**
+   * Removes the variable named {@link name} from this configuration.
+   *
+   * Returns whether thee variable was removed.
```
Comment (Goodwine) [inline pkg/sass-parser/lib/src/configuration.ts:120]:
```suggestion
   * Returns whether the variable was removed.
```

**Review by Goodwine — APPROVED** [review]

**Pull request merged**